README-sitedev.md

```diff
@@ -77,9 +77,9 @@ jena is at https://archive.apache.org/dist/jena/binaries/apache-jena-3.10.0.tar.
 
 The dependencies should be visible in the [Makefile](Makefile). The basic idea is:
 
- * [ontology/](ontology/)*.md  --[extract yaml]--> [registry/ontologies.yaml](registry/ontologies.yaml) --> [_config.yml](_config.yml)
+ * [ontology/](ontology/)*.md  --[extract yaml]--> [registry/ontologies.yml](registry/ontologies.yml) --> [_config.yml](_config.yml)
 
-[registry/ontologies.yaml](registry/ontologies.yaml) is also used to
+[registry/ontologies.yml](registry/ontologies.yml) is also used to
 create [registry/ontologies.rdf](registry/ontologies.rdf) via a
```

Comment:
there are several RDF files already available:

1. ontologies.nt  (in the N-tuples format)
2. ontologies.ttl (in the turtle format)
3. ontologies.jsonld (in the JSON-LD format)

Even though content may be RDF, `*.rdf` typically isn't used as a file extension. You can list these three as an alternative to finish this up

Reply:
Thanks, Charlie! See if my edit is what you intended.

BTW, I noticed that this documentation refers to the "prototype" obofoundry web page -- aren't we past the "prototype" phase at this point?

Reply:
you're welcome!

Reply:
> not sure if I can comment on the prototype status of OBO Foundry

I hope you said that @cthoyt with a slight smile and intellectual curiosity, not with hidden undertone :P